BnbnavNetClient/ViewModels/MainViewModel.cs

```diff
@@ -12,6 +12,9 @@ public sealed class MainViewModel : ViewModel
     [Reactive]
     public bool EditModeEnabled { get; set; }
 
+    [Reactive]
+    public bool HighlightTurnRestrictionsEnabled { get; set; }
```

Comment:
Maybe we can use an enum with all the types of spy mode we have instead? We'll be having lots of spy modes in the future

Reply:
Would they be mutually exclusive though?

Reply:
Yeah, in the current bnbnav they are mutually exclusive, although maybe we can use bitflags if they are not?

Reply:
tbh i don't see why they should be mutually exclusive, the highlights can be different colors/styles so that you can see them all if all of them are enabled

Reply:
I mean if they're not exclusive then I think we should just keep them as separate `bool`s; We can move them out into a separate object if needed, but it's probably easier tan implementing all the events for the bitflags

Reply:
I think we can sort this out in the future